script/lib/config.py

```diff
@@ -11,6 +11,7 @@
     'cygwin': 'win32',
     'darwin': 'darwin',
     'linux2': 'linux',
```

Comment:
what's `linux2`? :)

Reply:
That's what python2 returns from `sys.platform`:

```
ipython2
Python 2.7.15 (default, Oct 15 2018, 15:26:09) 

In [1]: import sys

In [2]: sys.platform
Out[2]: 'linux2'
```